student/src/main/webapp/Pages/systemdiagnostics.xhtml

```diff
@@ -14,7 +14,6 @@
 	<ui:define name="headerHolder">
 		<tds:ResourcesLink file="~/Scripts/scripts_diagnostics.xml" name="diagnostictool" />
 		<tds:GlobalJavascript contextName="DiagnosticShell" />
-		<script src="#{request.contextPath}/Scripts/Blackbox/blackbox_messages_#{diagnosticsBackingBean.loginClientName}.js" type="text/javascript" />

```

Comment:
I'm not sure why this was originally clobbering translations from the database in favor of hard-coded source-provided message key/values... but I removed it so the translations could be provided via the database.